test/BackendGeb/Eval/Base.hs

```diff
@@ -63,8 +63,8 @@ gebEvalAssertion' _mainFile expectedFile step gebMorphism = do
 gebEvalErrorAssertion :: Path Abs File -> (String -> IO ()) -> Assertion
 gebEvalErrorAssertion mainFile step = do
   step "Parse"
-  input <- readFile (toFilePath mainFile)
-  case Geb.runParser mainFile input of
+  input' <- readFile (toFilePath mainFile)
+  case Geb.runParser mainFile input' of
```

Comment:
Name change

Reply:
This change was moved to https://github.com/anoma/juvix/pull/2582